sdk/schemaregistry/schema-registry-json/samples-dev/withEventHubsConsumerClient.ts

```diff
@@ -114,7 +113,8 @@ export async function main() {
         console.log(`Error on partition "${context.partitionId}": ${err}`);
       },
     },
-    { startPosition: earliestEventPosition }
+    // Set the skipParsingBodyAsJson option to disable automatic parsing to Json object
```

Comment:
nitpick: minor edit:
```suggestion
    // Set the skipParsingBodyAsJson option to disable automatic JSON parsing of the message so we can deserialize it with the JSON serializer instead.
```